src/derivative_wrappers.jl

```diff
@@ -3,7 +3,7 @@ struct SensitivityAlg{CS,AD,FDT} <: DiffEqBase.DEAlgorithm
   quad::Bool
   backsolve::Bool
 end
-Base.@pure function SensitivityAlg(;chunk_size=0,autodiff=true,diff_type=Val{:central},autojacvec=autodiff,quad=true,backsolve=false)
+Base.@pure function SensitivityAlg(;chunk_size=0,autodiff=true,diff_type=Val{:central},autojacvec=autodiff,quad=false,backsolve=true)
```

Comment:
`backsolve` can't be the default because it's not technically correct. It's unstable. Besides, no one (except ML people as of 1 month ago) would expect it to be this version.

Reply:
Sure. What about `quad` – is there any reason to keep that around at this point?